compiler/vertex-meta_op_base.h

```diff
@@ -232,6 +187,51 @@ class vertex_inner<meta_op_base> {
              get_string_def(lhs) == get_string_def(rhs) &&
              std::equal(lhs->begin(), lhs->end(), rhs->begin(), rhs->end(), deep_equal);
     }
+
```

Comment:
What's the point of this change? (moving a group of functions)? To make them public?

Reply:
Yes, I used these methods to calculate the size of the vertex.
See `create_vertex_with_string` implementation.